react.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+  
+<head>
+  <title>1React-app</title>
+  <link rel="stylesheet" type="text/css" href="theme.css">
+  <script src="https://fb.me/react-15.1.0.js"></script>
+  <script src="https://fb.me/react-dom-15.1.0.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-core/5.8.23/browser.min.js"></script>
+  <script src="https://npmcdn.com/react-router@2.4.0/umd/ReactRouter.min.js"></script>
+
+  <head>
+  	<body>
+  		<div id="container">
+  			
+  		</div>
+  		<script type="text/babel">
+  			var { Router, Route, IndexRoute, IndexLink, Link } = ReactRouter;
+  			var destination = document.querySelector("#container");
+
+  			var Home = React.createClass({
+  				render: function() {
+  					return (
+  						<div>
+  							<h2>Hello</h2>
+  							<p>The cat sat on the roof and the dog was smelly~</p>
+  							<p>Beer and pizza are an american combo</p>
+  						</div>
+  					);
+  				}
+  			});
+
+  			var Contact = React.createClass({
+  				render: function() {
+  					return (
+  						<div>
+  							<h2>Got Questions?</h2>
+  							<p>Just post on our forms <a href="http://forum.kirupa.com">forum</a></p>
+  						</div>
+  					);
+  				}
+  			});
+
+  			var Stuff = React.createClass({
+  				render: function() {
+  					return (
+  						<div>
+  							<h2>Stuff</h2>
+  							<p>Turltes</p>
+  							<ol>
+						        <li>1 turtle</li>
+						        <li>2 turtle</li>
+						        <li>3 turtle</li>
+						        <li>4 turtle</li>
+					            <li>More turtle</li>
```

Comment:
Well it is more of a turtle best practice than a react best practice. It just seems weird that you would only have 4 turtles, when you could have a lot more, you know?